packages/storage/src/common/urls.ts

```diff
@@ -25,6 +25,11 @@ export const DEFAULT_GATEWAY_URLS: GatewayUrls = {
  */
 export const TW_IPFS_SERVER_URL = "https://upload.nftlabs.co";
```

Comment:
Can we then remove this const and the PINATA_IPFS_URL below?

Reply:
Not sure if this is used externally is the reason I didn't use it. I have a vague memory of some other places using it outside the storage SDK (unsure if it's accurate).

Reply:
Just searching in the whole repo should let you know right? I don't think we use it anywhere else